tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "target": "esnext" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */,
+    "module": "commonjs" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
+    "lib": [
+      "es6"
+    ] /* Specify library files to be included in the compilation. */,
+    "skipLibCheck": true,
+    "allowJs": true /* Allow javascript files to be compiled. */,
+    "checkJs": false /* Report errors in .js files. */,
+    "jsx": "react-native" /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */,
+    "noEmit": true /* Do not emit outputs. */,
```

Comment:
I think the TypeScript should be compiled and that the javascript output is what should be published to NPM in the future - so I'm guessing this flag should be `false` and that we should set up a `lib` folder to compile to and publish from.

@andrekovac your input is much appreciated here 💫

Reply:
Thanks for the info! @JonasWho shared the article https://rossbulat.medium.com/how-to-develop-typescript-npm-packages-for-react-native-a00f88f8a96c with me which you shared with him. In this article I also learned about this best practice.